docs/source/deployment/databricks/choosing_databricks_workflow.md

```diff
@@ -0,0 +1,35 @@
+# Choosing the Right Kedro Workflow for Databricks
+
+Databricks offers integration with Kedro through three principal workflows: The Databricks Deployment Workflow, the Databricks Development Workflow, and the Workflow for Developing Kedro projects on Databricks using notebooks. To enhance your development experience, it is crucial to choose the workflow that best fits your project's needs. Let's break down the advantages and use-cases of each workflow to help you make an informed decision.
+
+## Databricks Deployment Workflow
+
+The Deployment Workflow is the go-to choice when dealing with complex project requirements that call for a high degree of structure and reproducibility. It's your best bet for a production setup given its support for CI/CD, automated/scheduled runs and other advanced use-cases. That being said, it might not be the ideal choice for projects requiring quick iterations due to its relatively rigid nature.
```

Comment:
```suggestion
This workflow is the go-to choice when dealing with complex project requirements that call for a high degree of structure and reproducibility. It's your best bet for a production setup given its support for CI/CD, automated/scheduled runs and other advanced use-cases. That being said, it might not be the ideal choice for projects requiring quick iterations due to its relatively rigid nature.
```